### ISSUE TITLE
ProCFW not properly hiding seplugins folder + DJMP Hot Tunes graphical issues

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuuhakuzx/procfw#238]
```
What is the expected output? What do you see instead?

DJ MAX Portable 3 - Does not start. Screen turns black, speakers make a silent 
low sound and the PSP turns off.

DJ Max Hot Tunes and DJ max Poertable Fever - starts normally, after playing 
one stage every next is off-sync all video, audio and gameplay go at diffirent 
variable speeds.

DJ MAX Portable International - Missing screens on some stages, does not play 
OST at all, game freezes after trying to play MV's.

Only DJ MAX Portable 2 seams to run properly without any glitches.

What version of the product are you using? On what operating system?

PSP 3004 4G with 6.20 PRO-B8
Tested with all 3 ISO modes bot with enabled and disabled ISO cache.
Region tested: Default, Korea, US, Europe
Other settings default.

Please provide any additional information below.


```

Original issue reported on code.google.com by `piotrekhenry` on 27 Aug 2011 at 11:01

**Comment by GoogleCodeExporter** [comment]
```
well...i'd say its the iso, because

djp3: proubley needs a patch by the sounds of it XD

the rest: proubley the iso, or the iso player, because some of these games 
won't really run properly unless you get a physical copy of it, i've have the 
physical copy of black square, and portable 1, and i've the digital copy bought 
straight from psn, i'd say try downloading a different iso or something, since 
i've learned that some of these isos that you find online arn't really fully 
complete... 
```

Original comment by `dudewith...@gmail.com` on 29 Aug 2011 at 1:16

**Comment by GoogleCodeExporter** [comment]
```
I tried both the patched and unpatched versions from diffirent locations. Gues 
it may require a diffirent patch for 6.20 or the default PRO iso loader might 
have some sunchronisation issues - thats my gues so far, but since there was 
some "DJMP3 patch" was mentioned in PRO-B6 changelog and it doesent work on 
B-7&8 i thought that mentioning and describing the issue might help in any way.
```

Original comment by `piotrekhenry` on 31 Aug 2011 at 10:56

**Comment by GoogleCodeExporter** [comment]
```
DJmax Portable 3 demo worked for me when 
Recovery menu/Hide CFW Files from game/Enabled
Tried that?

```

Original comment by `morrn...@gmail.com` on 13 Sep 2011 at 5:05

**Comment by GoogleCodeExporter** [comment]
```
Yes. It's Enabled by default.
```

Original comment by `piotrekhenry` on 13 Sep 2011 at 10:41

**Comment by GoogleCodeExporter** [comment]
```
Ok getting a little tired of saying this but update to PRO B9

don't get me wrong just a little tired that is all and see if it is fixed :)
```

Original comment by `randqalan@gmail.com` on 13 Sep 2011 at 10:50

**Comment by GoogleCodeExporter** [comment]
```
I did update to PRO B9 (not nightly). It's not fixed yet.
```

Original comment by `piotrekhenry` on 13 Sep 2011 at 11:01

**Comment by GoogleCodeExporter** [comment]
```
Try turning off iso caching to see if it will fix it I have been noticing alot 
of games get fixed by turning it off
```

Original comment by `randqalan@gmail.com` on 13 Sep 2011 at 11:04

**Comment by GoogleCodeExporter** [comment]
```
Like I wrote in the issue above - I had tested it both with Disabled ISO Cache 
and with Enabled ISO Cache with default settings.

Just tested it AGAIN on PRO B9 (not nightly) just to make sure. Still the same 
issue.

DJMP3 Game iso works fine on my friends PSP2K with 5.xx cfw so the iso is not 
damaged.
```

Original comment by `piotrekhenry` on 13 Sep 2011 at 12:05

**Comment by GoogleCodeExporter** [comment]
```
Just tested it on todays 6.20 Nightly build and it still doesent work.
```

Original comment by `piotrekhenry` on 13 Sep 2011 at 12:43

**Comment by GoogleCodeExporter** [comment]
```
One more suggestion then I give up have you tried other drivers besides inferno 
```

Original comment by `randqalan@gmail.com` on 13 Sep 2011 at 12:47

**Comment by GoogleCodeExporter** [comment]
```
Yes I have tried Inferno, M33 driver and Sony NP9660 (also with ISO catche 
enabled and disabled).

It's not urgent. Maby while fixing other issues this issue will get solved as 
well.


```

Original comment by `piotrekhenry` on 13 Sep 2011 at 1:27

**Comment by GoogleCodeExporter** [comment]
```
I downloaded the demo of DJMP3 and tried to run it. Itdoesent work too, but I 
can provide you with some new info.
The DJMP3 Demo starts, shows the screen with "please read the user manual[...]" 
notice and returns an error: "The Game could not be started. (80010087)"
Maby the same (or similar) error occurs with the full version but it doesent 
show up before the PSP shuts down.
```

Original comment by `piotrekhenry` on 13 Sep 2011 at 3:28

**Comment by GoogleCodeExporter** [comment]
```
Try disabling all plugins?
```

Original comment by `morrn...@gmail.com` on 12 Oct 2011 at 11:46

**Comment by GoogleCodeExporter** [comment]
```
I didnt enable any additional plugins.
```

Original comment by `piotrekhenry` on 12 Oct 2011 at 7:53

**Comment by GoogleCodeExporter** [comment]
```
Updated to 6.20 PRO-B10.fix1

DJMP International Works fine
DJMP 2 Works Fine
DJMP Fever Works fine

DJMP Hot Tunes still doesent work properly. The same issues occur. After doing 
some tests on other CFW (5.xx m33-6) it seams this is a common issue for almost 
every CFW so I gues it's not a matter of firmware issues.

DJMP 3 still doesent work. Hardware same as before (first post), 6.20 
PRO-B10.fix1, default configuration, no additional plugins enabled. ISO of DJMP 
3 (patched) was working on PSP2K with 5.xx m33-6.
```

Original comment by `piotrekhenry` on 1 Nov 2011 at 7:50

**Comment by GoogleCodeExporter** [comment]
```
I havent tested DJMP Black Square and DJMP Clazziquai.
```

Original comment by `piotrekhenry` on 1 Nov 2011 at 7:52

**Comment by GoogleCodeExporter** [comment]
```
I only have DJMP3, Hot Tunes, and Black Square, and all three work flawlessly 
on my Go.
Hot Tunes and Black Square worked from the get-go, but I had to remove my 
SEPLUGINS folder from my PSP first before starting Portable 3. It ran, and I 
disabled-enabled the [hide CFW folder from game] in the Recovery Menu, reset my 
device, and put back SEPLUGINS. It started working after that, even if I had 
the folder in my PSP, which I heard checks for that folder and refuses to boot. 
Maybe it has something to do with my save being there already, but it's worth a 
shot.

Worked on B9, B10, and B10.fix.
```

Original comment by `zygi...@gmail.com` on 4 Nov 2011 at 9:57

**Comment by GoogleCodeExporter** [comment]
```
I'll try that. would you mind shareing the location you got your Hot Tunes copy 
from ? I tried multiple diffirent ISO's, fixed and raw, but they all didnt 
working properly.
```

Original comment by `piotrekhenry` on 4 Nov 2011 at 11:47

**Comment by GoogleCodeExporter** [comment]
```
MAGIC! That solution DID WORK for the DJMP3. Great thanks. 
```

Original comment by `piotrekhenry` on 4 Nov 2011 at 2:56

**Comment by GoogleCodeExporter** [comment]
```
As for the DJMP Hot Tunes this solution kind of improved stability, but still 
some glitches occur.

I deleted the SEPLUGINS folder, disabled and enabled again the "Hide CFW files 
from game" option, started the game (Tried Inferno and Sony NP9660 ISO Drivers) 
and after about 3 completed songs multiple graphical glitches occur. Everything 
starts to blink (bacground video, interface) but at least the music is still in 
sync with the rest unlike before.

Personal note: I'm really amazed that disabling all the plugins from seplugins 
folder was not enough and that only removing the folder from the MS worked for 
DJMP3 and improved DJMP HT's stability.
```

Original comment by `piotrekhenry` on 4 Nov 2011 at 3:55

**Comment by GoogleCodeExporter** [comment]
```
You could put back the folder if you want to try, I put it back and tried to 
use cheats and they worked.

As for Hot Tunes, check your e-mail for more assistance. I personally 
encountered that bug with Fever, but tried a different ISO and the glitches 
disappeared. Either you're really unfortunate with your sources or it's a 
conspiracy against you.

So far I've played the International Edition (1), Clazziquai, Black Square, Hot 
Tunes, Fever, and 3 without a hitch, maybe except for a few performance issues 
with my Fever copy. No more glitches and unsync'd music.
```

Original comment by `zygi...@gmail.com` on 8 Nov 2011 at 4:54

**Comment by GoogleCodeExporter** [comment]
```
I know I can put the seplugins folder back after the first run of DJMP3. DJMP 
Hot Tunes from your source works perfect. Gues I was really unlucky with the 
ISOs I found.

Great thanks. That was a lot of help. Everything should be fine now.
```

Original comment by `piotrekhenry` on 9 Nov 2011 at 5:31

**Comment by GoogleCodeExporter** [comment]
```
I found a completely new issue in DJMP Hot Tunes.

What steps will reproduce the problem?
1. Start the game.
2. Choose 6Buttons Mode (Hard Mode)
3. Start playing "Sunny Side" track.

What is the expected output? What do you see instead?
I should be able to play the trask to the end. Instead the game freezes after 
the 29th note. Music stil plays, everything else freezes (background video , 
user interface), home button responds, but doesent exit to XMB, requires a 
manual hard reset of the device.

What version of the product are you using? On what operating system?
PSP 3004 03G with 6.20 PRO-B10.fix1 with permanent patch

Please provide any additional information below.
I tested this with an ISO that was free from all the proeviously listed issues.
Treid using Inferno, Sony NP9660 (both with Enabled and Disabled ISO Cache) and 
M33 Driver. All 5 tests gave the same result.
All aditional plugins disabled. Tested with and without seplugins folder 
(deleted it).
Other settings default.
```

Original comment by `piotrekhenry` on 9 Nov 2011 at 9:14

**Comment by GoogleCodeExporter** [comment]
```
I have the exact same issue, but further research (Google) finds that it exists 
even in the UMD game, hence an official bug. Apparently it's fixed if you 
change your PSP's language to Japanese.
```

Original comment by `zygi...@gmail.com` on 10 Nov 2011 at 7:33

**Comment by GoogleCodeExporter** [comment]
```
Changing the language to Japanese did help.
Thanks again. I hope I wont find any other issues to post in this thread.

PS: The developers, or maby more like the beta testers failed big time ... 
missing a bug that a casual player found in so little time. Also not releasing 
a fixed version of the game in future releases was another failure from the 
developers side. I didnt think that such bug could be official and not cfw 
related - after all most of the issues concerning runing diffirent games from 
iso's is cfw related.
```

Original comment by `piotrekhenry` on 10 Nov 2011 at 8:34

**Comment by GoogleCodeExporter** [comment]
```
It's not the only time, the first release of Black Square has a huge bug in 
where you can only see 2 songs (initially) for the 6B Arcade mode, and will 
only show other songs you unlock, never the songs that are already available. 
Clazziquai has issues in where you can't play certain unlocked songs in Arcade 
mode, and 3 has some technical issues as well. All have been patched, but 
that's past release so I get what you mean.

Hope everything will run fine for you from now on. Haha.
```

Original comment by `zygi...@gmail.com` on 10 Nov 2011 at 8:40

**Comment by GoogleCodeExporter** [comment]
```
While playing DJMP3 I noticed that while playing 4.2 mode In some of the songs 
while you have to play on the left or right track - instead of some mixing 
sounds a fragment of a diffirent song is played. Is it suposed to be like that 
or is that another bug ?
```

Original comment by `piotrekhenry` on 10 Nov 2011 at 10:01

**Comment by GoogleCodeExporter** [comment]
```
It is supposed to be like that.

http://www.cyphergate.net/wiki/index.php?title=DJMAX_Portable_3#Game_Tracklist
```

Original comment by `catnip...@gmail.com` on 10 Nov 2011 at 11:19

**Comment by GoogleCodeExporter** [comment]
```
DJ MAX Portable 3 - Does not start. Screen turns black, speakers make a silent 
low sound and the PSP turns off.
The only way to start the game is stand-by the PSP right after the gameboot 
logo and turn on again.

PSP Go 5G with 6.60 PRO-B10.fix1
Tested with all 3 ISO modes boot with enabled and disabled ISO cache.
Tested with CFW Files Hide option enabled and disabled.
Region tested: Default, Korea, US, Europe
Other settings default.


```

Original comment by `mikemarcio@hotmail.com` on 10 Nov 2011 at 5:01

**Comment by GoogleCodeExporter** [comment]
```
There's an easier solution above that doesn't require the sleep-trick.
```

Original comment by `zygi...@gmail.com` on 13 Nov 2011 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
Hi there,

The solution above does not work for me.

Thanks.
```

Original comment by `mikemarcio@hotmail.com` on 13 Nov 2011 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
Running 6.60 PRO-B9 on PSP GO (5g). DJ Max P. 3 was making PSP turn off after a 
couple of seconds. I've tried using the fix you mentioned to play DJ Max 
Portable 3, got it to work but only partially works. I can play without 
problems for the first 7 mins (about 2 songs in arcade mode), but then the game 
gets distorted and becomes unplayable. I am using CTF themes with CXMB, should 
I switch this off? Anyone know a fix for this? Anyone else having this problem?
```

Original comment by `dragonlo...@gmail.com` on 15 Apr 2012 at 1:23

**Comment by GoogleCodeExporter** [comment]
```
run with "Hide CFW files from game" option and delete the SEPLUGINS folder on 
your MS. It should work fine. 

But I do hope that PRO supports custom directory for SEPLUGINS. It might fix 
this DJMP issues when SEPLUGINS is detected.

BTW: DJMP Black Square KOREAN version wont work and deletes the iso file 
contained on your ISO folder, even with the "Hide CFW files from game" enable. 
ALl you have to do is to sleep your PSP right after the PSP Boot up screen when 
you start a game. Also delete the SEPLUGIN folder to make it glitchless, since 
one SEPLUGINS is there, the game start to get glitch like random graphics 
glitch, weird sound tempo, game become too slow, et..
```

Original comment by `remleika...@gmail.com` on 15 Apr 2012 at 11:10

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reply, but got it working with the SEPLUGINS folder and have CXMB 
(custom themes) enabled on my PSP GO. All the DJ Max Portable games work fine 
(aside from the official bugs). Just got the file from another source (5.00 M33 
-Gen compatible). Good luck all.
```

Original comment by `dragonlo...@gmail.com` on 17 Apr 2012 at 9:23

**Comment by GoogleCodeExporter** [comment]
```
The issue was fixed by "Hide cfw files in game"
```

Original comment by `devnonam...@gmail.com` on 15 Jun 2012 at 3:12
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
DJMAX Portable 3 still hangs my PSPgo for some reason... =S
```

Original comment by `mikemarcio@hotmail.com` on 16 Jun 2012 at 2:34

**Comment by GoogleCodeExporter** [comment]
```
Seems like this is partially fixed.

Please try with a fresh dumped version with no patch: those from internet do 
have embedded patches that sometimes cause some issues on pro custom firmwares.


"All the DJ Max Portable games work fine"
Please guys confirm that with your own dumped version with the following 
settings:
-plugins disabled
-iso cache disabled
-high memory in retail games disabled
-memory stick speedup disabled

```

Original comment by `devnonam...@gmail.com` on 23 Jun 2012 at 1:28
- Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
```
Even if all of them work, some of the troubleshooting stuff like removing the 
seplugins folder for the 1st run of DJMP 3 to work or changing the Language to 
Japanese for DJMP Hot Tunes to work properly should be left for future users to 
read as a tutorial to make their copy of the game work.
```

Original comment by `piotrekhenry` on 24 Jun 2012 at 5:32

**Comment by GoogleCodeExporter** [comment]
```
I myself don't have nor haad any problem with HotTunes, aside from the language 
of the PSPgo MUST BE in English for the game to work. About the DJMP3, will 
test soon and post result in here.
```

Original comment by `mikemarcio@hotmail.com` on 24 Jun 2012 at 11:26

**Comment by GoogleCodeExporter** [comment]
```
Just tried DJMAX Portable 3 USA version, no seplugins folder, plugins disabled, 
ISO cache disabled, Memory Stick Speedup disabled, Hide CFW enabled.
The ISO is UNTOUCHED! ID: ULUS-10538. I load the game, PSP animation OK, after 
that, black screen, freezes PSPgo, and PSPgo shuts itself off.
FW Installed on PSPgo: 6.60 with 6.60 PRO-B10 fix1 has Fast Recovery.
Using Internal Memory of PSPgo.
If more details needed, just comment. I really would like this issue fixed. =)
Thanks.
```

Original comment by `mikemarcio@hotmail.com` on 24 Jun 2012 at 5:57

**Comment by GoogleCodeExporter** [comment]
```
New test: DJMax Portable 3.
Usign Stand-by after PSP logo method.
UNTOUCHED USA ISO.

Using this method, game boots, but after certain time of gameplay, screen 
starts to flicker and sound samples get all messed up.
```

Original comment by `mikemarcio@hotmail.com` on 30 Jun 2012 at 3:14

**Comment by GoogleCodeExporter** [comment]
```
Probably a damadged ISO. That was the issu in my case.
```

Original comment by `piotrekhenry` on 2 Jul 2012 at 4:56

**Comment by GoogleCodeExporter** [comment]
```
My ISOs are NOT damaged.
It's impossible to several UNTOUCHED ISOs be damage.
I tried with KOR, JPN and USA version. All same issue.
```

Original comment by `mikemarcio@hotmail.com` on 2 Jul 2012 at 5:38

**Comment by GoogleCodeExporter** [comment]
```
Another test:

Grabbed myself a patched ISO this time (runs OK in my 2000 5.50GEN).
Opened the ISO with WinHEX, changed the ISO and SEPLUGINS values of the 
decrypted ELF to IS0 (that's a zero) and S3PLUG1NS. The ISO is patched with 
prometheus prx.
Still, the game does not run on my Go with 6.60 PRO-B10.fix1.
```

Original comment by `mikemarcio@hotmail.com` on 3 Jul 2012 at 1:20

**Comment by GoogleCodeExporter** [comment]
```
Did you rip the ISO's yourself ? If not you cant be certain that the ISO's you 
found "somewhere in the internet" arent modified or damaged. From my experience 
the same damadged Iso of Hot Tunes was copied into multiple PSP ISO Providing 
sites P2P software etc. Even tho a lot of users were leaving comments that it's 
broken, nobody fixed that and the plague of damadged ISO's is spreading.
```

Original comment by `piotrekhenry` on 6 Jul 2012 at 12:12

**Comment by GoogleCodeExporter** [comment]
```
SCENE ISOs are always PROPER, else they are marked has NUKED.
I never grab my stuff from "around the web". I only use PROPER SCENE releases.
My ISO from HotTunes is 100% OK. And the ones from DJMAX3 are also 100% OK.
How do I know? Because they work OK on 5.50 GEN-D on my 2000 PSP.
```

Original comment by `mikemarcio@hotmail.com` on 6 Jul 2012 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
OK, I have an idea. =)
Why don't you share your ISO with me, I give it a test on my PSPgo, and we'll 
see the results. My e-mail: m(at)kof(dot)zzn(dot)com
(thumbs up)
```

Original comment by `mikemarcio@hotmail.com` on 6 Jul 2012 at 1:09

**Comment by GoogleCodeExporter** [comment]
```
After a long extensive research I finally found the problem on the PSPgo 
systems.

First: a big special thanks to piotrekhenry for sharing is information and ISO 
file with me.

Here's the solution.
In PSPgo: THE ISO FILE MUST BE RUN FROM MEMORY STICK.

If you try to run the game ISO from the internal memory, the PSP freezes at 
boot.
The only way to boot the game from internal memory, and without freeze, is to 
do the stand-by trick right after the PSP animation boot, and the ISO must be 
patched (Kiss of Death patch is OK) so that the game does not start to flicker 
and have sond problems in middle-game.

So here you have it. For those in PSPgo's.

Internal Memory: Must perform stand-by trick no matter what.
Memory Stick: Everything runs OK.

In both cases the ISO file must be patched with Kiss of Death patch to run 
properly without flickering issues and sound problems.
```

Original comment by `mikemarcio@hotmail.com` on 9 Jul 2012 at 5:20

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem with my iso of DJMax Portable 3 (JAP). I can play two 
songs perfectly, but in the middle of the third, the screen and sound begin to 
bug until the screen turns completely black.
My PSP-3000 runs CFW 6.60 PRO-B10.

I already try to delete seplugins folder, change language in japanese, nothing 
has worked so far.
I also tried different isos of the game, always the same issue.

Thank you so much for your help!
```

Original comment by `Vinc.Ri...@gmail.com` on 2 Sep 2012 at 10:10

**Comment by GoogleCodeExporter** [comment]
```
@ Vinci Ricci

Read the solution that's provided by MDashKusanagi. The solution is the 
specific patch for the Japanese version; use the latest iso tool to patch the 
game since iso tool does include the patch for the Japanese DJMax3, but not for 
other regions.
```

Original comment by `cmtang...@gmail.com` on 10 Sep 2012 at 9:23

**Comment by GoogleCodeExporter** [comment]
```
you can use DJMP3 PSN version and convert it to ISO with a PC decryption tool 
(not going to tell that). After that, make sure that the cfwblocker plugin is 
activated (take note, PRO's built-in CFW blocker doesnt even work) and run the 
game as normal.

Clazziquai and Black Square needs patching (or rather game updates need to be 
installed) to avoid them getting a sound problem, KOD patch is fine for this 
but installing the update is much more preferred. Also dont forget to activate 
the cfwblocker plugin or your ISO and save files will be gone/deleted for good.
```

Original comment by `remleika...@gmail.com` on 27 Jan 2013 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
@remleikawura

Your Method really works.
Thank you very much!
```

Original comment by `arn...@gmail.com` on 21 Feb 2013 at 5:10

**Comment by GoogleCodeExporter** [comment]
```
Mine loads all DJMAX Portable games...

...except Fever and Hot Tunes, as I played for a few songs the graphic 
artifacts shows up and the music/bg music tends to speed/pitch up and down 
until it hangs.

As for DJMAX Portable 3, I got the very same black screen problem. Actually, 
after the PSP logo screen it remains black for a few seconds then the PSP shuts 
down. I thought it was the Pro CFW (I'm using a 6.20 Pro-B10, btw), but I kinda 
think it was my MS Pro Duo. So I backed up all of my files from it, format it 
and put them back in. Then it worked and DJMAX Portable 3 loads fine. When I 
finally purchased a new 8GB MS Pro Duo, the very same problem happened again. 
But I tried removing the DJMAX Portable 3 save folder instead of doing the 
usual long routine and let the game load. It worked again! But since I removed 
the save folder, I put it back again, and it worked just fine.

So I suggest you remove the DJMAX Portable 3 save folder temporarily (if you 
have already got a lot of unlocks and achievements, otherwise remove it 
entirely), and see if it works fine.
```

Original comment by `djash1...@gmail.com` on 9 Apr 2013 at 7:15

**Comment by GoogleCodeExporter** [comment]
```
Hi so far I've read all the comments here. I have problem running just DJMAX 
Black Square. Fever and Clazziquai works perfectly fine. I don't know why my 
Black square goes glitchy and blink black screen. I know some of you guys 
mentioned about seplugins and what not. Could someone help me with the problem 
I am having? Thank you. 

I have psp-3001 with 6.60 RPO
```

Original comment by `jungha1...@gmail.com` on 12 Jul 2013 at 9:21

**Comment by GoogleCodeExporter** [comment]
```
Issue 573 has been merged into this issue.
```

Original comment by `devnonam...@gmail.com` on 25 Aug 2013 at 2:01

**Comment by GoogleCodeExporter** [comment]
```
Someone to confirm that the DJMP Hot Tunes graphical issues don't happen on OFW?
```

Original comment by `devnonam...@gmail.com` on 26 Aug 2013 at 11:45
- Changed title: **ProCFW not properly hiding seplugins folder + DJMP Hot Tunes graphical issues**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
DJMP Hot tunes graphical issues dont occur on a properly ripped .iso nor in OFW.
The problem is that most of the sources of pirated copy of this game has a copy 
of the same broken .iso.
```

Original comment by `piotrekhenry` on 28 Aug 2013 at 5:02

**Comment by GoogleCodeExporter** [comment]
```
Because it's impossible to apply the PSN patch to .iso it is obligatory to 
change the PSP's language to Japanese before starting the game.
```

Original comment by `piotrekhenry` on 28 Aug 2013 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
Issue 538 has been merged into this issue.
```

Original comment by `devnonam...@gmail.com` on 30 Aug 2013 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
Issue 537 has been merged into this issue.
```

Original comment by `devnonam...@gmail.com` on 30 Aug 2013 at 9:16

**Comment by GoogleCodeExporter** [comment]
```
Issue 536 has been merged into this issue.
```

Original comment by `devnonam...@gmail.com` on 30 Aug 2013 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
Issue 526 has been merged into this issue.
```

Original comment by `devnonam...@gmail.com` on 30 Aug 2013 at 9:53

**Comment by GoogleCodeExporter** [comment]
```
I've been trying all these fixes, along with ones from other sources to make my 
DJ MAX Hot Tunes .iso work. I've tried several different .isos on both 5.50 
Prome4 and recently 6.60 Pro C2. None of them have worked so far. The best 
shape I've gotten the game to run in is perfectly for about 3 songs and then 
either graphical or audio distortion afterwards.

I was wondering if anyone could point me in the right direction of a "clean" or 
"untouched" iso, or help me fix the one that I have.

PS Black Square, Clazziquai and Portable 2 run perfectly on my PSP-2000.

Thanks.
```

Original comment by `derpolog...@gmail.com` on 12 Sep 2013 at 4:50

**Comment by GoogleCodeExporter** [comment]
```
This error happens ONLY on all the pirated copies of ONE broken ISO distributed 
via torrents and other warez/P2P means. A properly ripped ISO from an UMD works 
fine.
```

Original comment by `piotrekhenry` on 17 Sep 2013 at 7:13

**Comment by GoogleCodeExporter** [comment]
```
We don't support piracy, moreover this issue is due to the fact you're using a 
patched ISO which seems corrupted.

Original issue has been fixed for a long time.
```

Original comment by `devnonam...@gmail.com` on 4 May 2014 at 12:21
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `piotrekhenry` on 23 Jan 2015 at 4:13
- Changed state: **Verified**
